src/main/java/internet/shop/controllers/shoppingcart/ShoppingCartController.java

```diff
@@ -11,16 +11,17 @@
 import javax.servlet.http.HttpServletResponse;
 
 public class ShoppingCartController extends HttpServlet {
```

Comment:
```suggestion
public class ShoppingCartController extends HttpServlet {
```
я б тут додав ще Get/ShowShoppnigCartController щоб неймінг був більш інформативним